ts_src/psbt.ts

```diff
@@ -659,25 +660,51 @@ export class Psbt {
     blindingPrivkeys: Buffer[],
     blindingPubkeys: Buffer[],
     opts?: RngOpts,
+  ): this {
+    return this.blindOutputsIndex(
+      blindingPrivkeys,
+      blindingPubkeys,
+      undefined,
+      opts,
+    );
+  }
+
+  blindOutputsIndex(
+    blindingPrivkeys: Buffer[],
+    blindingPubkeys: Buffer[],
+    outputsIndexToBlind?: number[],
```

Comment:
I think this name can be simplified
```suggestion
    outputIndexes?: number[],
```